src/components/pages/batch/list/index.tsx

```diff
@@ -94,36 +91,57 @@ function BatchListPageComponent() {
       >
         Create Lot
       </Button>
-      <Button
-        colorScheme="blue"
-        variant="solid"
-        onClick={handleOnCreateBatch}
-        isDisabled={!hasAccess([ROLES.ADMIN, ROLES.COOPERATIVE, ROLES.COLLECTION_CENTER], user)}
-        leftIcon={<AddIcon />}
-      >
-        Create Batch
-      </Button>
     </ButtonGroup>
   );
 
   const onBatchUpdate = (props) => {
     onToggle();
     actions.updateBatch(props);
+    setTriggerRender(!triggerRender);
   };
 
+  // Generate dynamic batchColumns based on state.batch
```

Comment:
remove this comment

Reply:
This comment is telling what below function is doing.